src/components/pages/AboutPage.js

```diff
@@ -1,7 +1,49 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/styles';
+import heroImage from 'images/Cover1.jpg';
 
-function AboutPage() {
-  return (<div />);
+export const styles = {
+  hero: {
+    backgroundImage: `url(${heroImage})`,
+    minHeight: 600,
+    backgroundAttachment: 'fixed',
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover',
+  },
+  heroText: {
+    position: 'absolute',
+    top: '0px',
+    left: '0px',
+    height: '600px',
```

Comment:
This 600 is the same value as the `minHeight` for the `hero` class, right? Let's extract this into a variable and use it in both places - take advantage of the power of JSS!

Reply:
great point, resolved